### PR TITLE
_headers & _redirects generation

### PR DESCRIPTION
**Pull request by developit**
Generate `_headers` and `_redirects` by passing assets through ejs templates.

Note: based against `service-worker` branch, wait to merge.

### CODE REVIEW

File: _headers.ejs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/
+  Cache-Control: nocache
```
Comment (jakearchibald) [inline _headers.ejs]: `no-cache`

File: _headers.ejs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+/serviceworker.js
+  Cache-Control: nocache
```
Comment (jakearchibald) [inline _headers.ejs]: `no-cache`

File: _redirects.ejs [hunk 3]
```diff
@@ -0,0 +1 @@
+/*    /index.html   200
```
Comment (jakearchibald) [inline _redirects.ejs]: Seems better to do:

```
/    /index.html
/*  / 307
```

That way we have a canonical URL for the app.

File: _headers.ejs [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  Cache-Control: max-age=31536000
+/:a.png
+  Cache-Control: max-age=31536000
+/:a.svg
+  Cache-Control: max-age=31536000
+
+# List of build assets at time of generation:
+<%_ for (let file of files) { _%>
+#  <%= file %>
+<%_ } _%>
```
Comment (jakearchibald) [inline _headers.ejs]: I guess we can remove the generated for now, until we need something more complicated?

File: _headers.ejs (hunk 2 above)
Comment (developit) [reply _headers.ejs]: ack.

File: _redirects.ejs (hunk 3 above)
Comment (developit) [reply _redirects.ejs]: this is the default behavior IIRC.

File: _headers.ejs (hunk 4 above)
Comment (developit) [reply _headers.ejs]: I was waiting until we determined if the URL patterns were functioning correctly.

**Comment by developit** [comment]
@jakearchibald up for re-review

**Comment by jakearchibald** [comment]
To be fair, if that rule was working, we'd redirect on all our subresources. I think I'll just remove the redirects & merge this.

**Comment by jakearchibald** [comment]
And that's the winner!

**Pull request merged**